### ISSUE TITLE
SDK autoconfigure

### ISSUE THREAD

**Issue by slinkydeveloper** [open-telemetry/opentelemetry-rust#890]
Hi all, I'm using [sdk autoconfigure](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure) on Java and I was wondering, is there something similar for the rust sdk? Is not, is there any plan/idea to implement it?

**Comment by lf-** [comment]
Some of these variables work as-is. It's a work in progress. See https://github.com/open-telemetry/opentelemetry-rust/issues/774

**Comment by jtescher** [comment]
Yeah the env config for each crate should be supported, there are some features like being able to select an exporter via variables that would require having all the exporters as dependencies. If someone wanted to maintain a new crate that provided that functionality it would also be possible.
